test/common/ssl/context_impl_test.cc

```diff
@@ -27,7 +27,10 @@ using testing::ReturnRef;
 namespace Envoy {
 namespace Ssl {
 
-class SslContextImplTest : public SslCertsTest {};
+class SslContextImplTest : public SslCertsTest {
+public:
```

Comment:
fwiw usually 'protected:' is used here as all the test methods are effectively subclasses. Just a suggestion.